boards/arm/frdm_k64f/frdm_k64f.dts

```diff
@@ -146,6 +146,26 @@ arduino_spi: &spi0 {
 	pinctrl-names = "default";
 };
 
+&spi1 {
+	status = "okay";
+	pinctrl-0 = <&spi1_default>;
+	pinctrl-names = "default";
+	cs-gpios = <&gpioe 4 GPIO_ACTIVE_LOW>;
+
+	sdhc0: sdhc@0 {
+		compatible = "zephyr,sdhc-spi-slot";
+		reg = <0>;
+		status = "okay";
+		label = "SDHC_0";
```

Comment:
[This broke main](https://github.com/zephyrproject-rtos/zephyr/runs/8239856646?check_suite_focus=true#step:11:22157):
> devicetree error: 'label' is marked as deprecated in 'properties:' in /__w/zephyr/zephyr/dts/bindings/sdhc/zephyr,sdhc-spi-slot.yaml for node /soc/spi@4002d000/sdhc@0.